styx-common/src/main/java/com/spotify/styx/storage/AggregateStorage.java

```diff
@@ -214,4 +214,9 @@ public void storeResource(Resource resource) throws IOException {
   public Optional<Backfill> backfill(String id) throws IOException {
     return datastoreStorage.getBackfill(id);
   }
+
+  @Override
+  public List<String> clientBlacklist() {
+    return datastoreStorage.clientBlacklist();
```

Comment:
If you don't have strong opinion, I would prefer runtime configuration. ;) hehehe